omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1404,12 +1415,14 @@ def paste_image(self, pil_img, img_name, panel, page, dpi):
         # in the folder to zip
         if self.zip_folder_name is not None:
             img_name = os.path.join(self.zip_folder_name, FINAL_DIR, img_name)
+            pil_img.save(img_name)
 
-        # Save Image to file, then bring into PDF
-        pil_img.save(img_name)
+        buffer = StringIO()
```

Comment:
will be better to start using ``io.StringIO`` if we want to start moving to python3